archivy/models.py

```diff
@@ -79,17 +68,16 @@ def process_bookmark_url(self):
         try:
             self.content = self.extract_content(parsed_html)
         except Exception:
-            flash(f"Could not extract content from {self.url}\n")
-            return
+            raise RuntimeError(f"Could not extract content from {self.url}\n")
```

Comment:
Makes more sense to keep the flash here so the user can see it, it's not a backend error it's the input that has a problem.